bdk-android/lib/src/androidTest/kotlin/org/bitcoindevkit/OfflineDescriptorTest.kt

```diff
@@ -0,0 +1,18 @@
+package org.bitcoindevkit
+
+import kotlin.test.Test
+import kotlin.test.assertTrue
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class OfflineDescriptorTest {
+    @Test
+    fun testDescriptorBip86() {
+        val mnemonic: Mnemonic = Mnemonic(WordCount.WORDS12)
```

Comment:
Instead of a random mnemonic should use a string of 12 words to test restoring and then can assert that we get back the exact correct descriptor.